.github/workflows/continuous-integration.yml

```diff
@@ -4,13 +4,10 @@ on:
   push:
     branches: [master]
   pull_request:
-    # The branches below must be a subset of the branches above
     branches: [master]
   schedule:
     - cron: '0 2 * * 1' # At 02:00 on Monday
 
-permissions: {}
```

Comment:
This is a security feature to restrict the permissions of the runner's GitHub token.

Reply:
No; By default, runners have read and write permissions. It's also configurable via https://github.com/loopbackio/loopback-next/settings/actions, but I'd rather not rely on that as it isn't as granular, and changes don't go through the pull request workflow.

Reply:
Ok I understand. A new thing to learn.